auth0/src/main/java/com/auth0/android/authentication/storage/ClockImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.auth0.android.authentication.storage;
+
+import com.auth0.android.util.Clock;
+
+/**
+ * Default Clock implementation used for verification.
+ *
+ * @see Clock
+ * <p>
+ * This class is thread-safe.
+ */
+final class ClockImpl implements Clock {
```

Comment:
no need to expose this implementation

Reply:
Some people use it, others prefer the `IClock` (interface) and `Clock` (implementation) variation. In this case, this implementation is internal so the pre/suffix doesn't really matter. For reference, in java-jwt we are doing the same https://github.com/auth0/java-jwt/blob/master/lib/src/main/java/com/auth0/jwt/ClockImpl.java.